infra_event_notifier/notifier.py

```diff
@@ -42,6 +42,6 @@ def notify(
             create_issue(
                 title=title,
                 text=text,
-                tags=tags,
+                labels=tags,
```

Comment:
jira doesn't support key:value tags. 

it supports labels, but we can create labels that look like tags ( example: `region:saas`)